src/lib/Scenes/Artwork/Components/ArtworkDetails.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Sans } from "@artsy/palette"
+import { ArtworkDetails_artwork } from "__generated__/ArtworkDetails_artwork.graphql"
+// import { capitalize } from "lodash"
+import React from "react"
+import { Button } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface ArtworkDetailsProps {
+  artwork: ArtworkDetails_artwork
+}
+
+interface ArtworkDetailsState {
+  show_all: boolean
+}
+
+export class ArtworkDetails extends React.Component<ArtworkDetailsProps, ArtworkDetailsState> {
+  constructor(props) {
```

Comment:
You can simplify this slightly by removing the constructor and just doing 

```
state = {
  showAll: false
}
```

([This article](https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599) talks through it more in depth)